### PR TITLE
test execution fixes

### PR DESCRIPTION
**Pull request by shahabhijeet**
Test execution enabled.
Disable test parallelization as it causes random failures

### CODE REVIEW

File: eng/mgmt/AzSdk.test.reference.props [hunk 1]
```diff
@@ -12,4 +12,7 @@
 		<DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.1" PrivateAssets="All" />
   </ItemGroup>
+  <ItemGroup> 
+	   <Compile Include="$(RepoRoot)\tools\DisableTestRunParallel.cs" Link="DisableTestRunParallel.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" /> 
+	</ItemGroup>
```
Comment (weshaggard) [inline eng/mgmt/AzSdk.test.reference.props]: nit: indention is off.

File: eng/mgmt/AzSdk.test.reference.props [hunk 2]
```diff
@@ -12,4 +12,7 @@
 		<DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.1" PrivateAssets="All" />
   </ItemGroup>
+  <ItemGroup> 
+	   <Compile Include="$(RepoRoot)\tools\DisableTestRunParallel.cs" Link="DisableTestRunParallel.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
```
Comment (weshaggard) [inline eng/mgmt/AzSdk.test.reference.props]: Do you know why the Exclude? It doesn't seem to be related?

Also at a future clean-up time we should move the DisableTestRunParallel.cs under the eng folder.

Comment (shahabhijeet) [reply eng/mgmt/AzSdk.test.reference.props]: I will do it right now, will move the disable*.cs to eng directory

I beleive this is one of remanants to previous project system, where linking was not working appropriately and had to exclude things. I will try removing it and if eveyrhing works, will remove the exclude clause

File: .azure-pipelines/mgmt.yml [hunk 3]
```diff
@@ -33,7 +33,7 @@ jobs:
       - task: MSBuild@1
         inputs:
           solution: build.proj
-          msbuildArguments: /t:RunTests /clp:ShowTimestamp $(LoggingArgs) /p:IgnorePathTokens="$(IgnoreBuildTokens)" /p:UseLegacyDirStructure=true /p:SkipTests=true
+          msbuildArguments: /t:RunTests /clp:ShowTimestamp $(LoggingArgs) /p:IgnorePathTokens="$(IgnoreBuildTokens) /p:SkipTests=true"
```
Comment (weshaggard) [inline .azure-pipelines/mgmt.yml]: Looks like the quoting is off here so SkipTests didn't correctly get set. Can I suggest this PR just undo all changes to the yml and just have the parallel stuff and then we can iterate on the yml and the rest of the test failures.

**Pull request merged**